processing/src/main/java/io/druid/segment/incremental/OffheapIncrementalIndex.java

```diff
@@ -92,6 +93,26 @@ public OffheapIncrementalIndex(
   }
 
   public OffheapIncrementalIndex(
```

Comment:
I have a similar constructor for `OnheapIncrementalIndex`, should I inline it as well?